supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+For more information, see the Microsoft document on link:https://docs.microsoft.com/en-us/windows/win32/ad/forests[Forests].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[active-directory-global-catalog]]
+==== image:images/yes.png[yes] Active Directory global catalog (noun)
+*Description*: The global catalog is a feature of Active Directory (AD) that allows a domain controller to provide information on any object in the forest, regardless of whether the object is a member of the domain controller’s domain. Domain controllers with the global catalog feature enabled are referred to as global catalog servers.  The global catalog provides a searchable catalog of all objects in every domain in a multi-domain Active Directory Domain Services (AD DS).
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:active-directory-forest[active-directory-forest]
+
+[discrete]
+[[active-directory-security-identifier]]
+==== image:images/yes.png[yes] Active Directory security identifier (noun)
+*Description*: A security identifier (SID) is a unique ID number assigned to an object in Active Directory, such as a user, group, or host. It is the functional equivalent of UIDs and GIDs in Linux.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[anaconda]]
+==== image:images/yes.png[yes] Anaconda (noun)
+*Description*: The operating system installer used in Fedora, Red Hat Enterprise Linux, and their derivatives. Anaconda is a set of Python modules and scripts with additional files like Gtk widgets (written in C), `systemd` units, and `dracut` libraries. Together, they form a tool that allows users to set parameters of the resulting (target) system.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[ansible-play]]
+==== image:images/yes.png[yes] Ansible play (noun)
+*Description*: Ansible plays are the building blocks of link:https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html#playbooks-intro[Ansible playbooks]. The goal of a play is to map a group of hosts to some well-defined roles, represented by Ansible tasks.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ansible-playbook[ansible-playbook]
+
+[discrete]
+[[ansible-playbook]]
+==== image:images/yes.png[yes] Ansible playbook (noun)
+*Description*: An Ansible playbook is a file that contains one or more Ansible plays. For more information, see the link:https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html#about-playbooks[official Ansible documentation about playbooks].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ansible-play[ansible-play]
+
+[discrete]
+[[ansible-playbook-sysroles]]
+==== image:images/yes.png[yes] Ansible playbook (noun)
```

Comment:
This looks identical to the above entry, just with a different description. Can you share why there are two? Does this same term have two different meanings? If so, I'd combine them into one entry, and just have the two different descriptions. A few other entries do this by using "(1)" and "(2)"

Reply:
I combined it together.